misk/src/test/kotlin/misk/web/interceptors/TracingInterceptorTest.kt

```diff
@@ -58,14 +98,45 @@ class TracingInterceptorTest {
     assertThat(tracingInterceptorFactory.create(TracingTestAction::call.asAction())).isNull()
   }
 
+  private fun get(path: String) {
+    val httpClient = OkHttpClient()
+    val request = okhttp3.Request.Builder()
+        .get()
+        .url(jettyService.httpServerUrl.newBuilder().encodedPath(path).build())
+        .build()
+    httpClient.newCall(request).execute()
+  }
+
   internal class TracingTestAction : WebAction {
     @Get("/trace")
     fun call(): Response<String> {
       return Response("success")
     }
   }
 
+  internal class FailedTracingTestAction : WebAction {
+    @Get("/failed_trace")
+    fun call(): Response<String> {
+      return Response("no good", statusCode = StatusCode.BAD_REQUEST.code)
+    }
+  }
+
+  internal class ExceptionThrowingTracingTestAction : WebAction {
```

Comment:
you still need a test case for a custom mapped exception

Reply:
Wait, what do you mean by custom mapped exception?  Some exception I've created that extends `ActionException`?

Regardless of how it's defined, I'm not sure how it matters. Converting of any exception into a response w/ >= 400 status code is the domain of the `ExceptionHandlerInterceptor`. This test demonstrates that if the `ExceptionHandlerInterceptor` does it's job properly, the `TracingInterceptor` will act accordingly (i.e. tag the span with an error tag).

Reply:
Yeah this is good enough

Reply:
ah good point. yea the `ActionException` still invokes the `ExceptionInterceptor`